src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -129,10 +154,31 @@ class ApplicationMenu extends Component<Props> {
       });
     }
 
+    // During onboarding, there is no selected project (because none exists
+    // yet). Therefore, we only want to show the 'Project' menu when a project
+    // is selected.
+    if (selectedProjectId && devServerTask) {
+      // The `Project` menu should be inserted right after `Edit`, which will
+      // have a different index depending on the platform.
+      const editMenuIndex = template.findIndex(menu => menu.id === 'edit');
+
+      template.splice(editMenuIndex, 0, {
+        id: 'project',
+        label: isMac ? 'Project' : '&Project',
+        submenu: [
+          {
+            label: isMac ? 'Clear Server Logs' : 'Clear server logs',
+            click: () => clearConsole(devServerTask),
+            accelerator: 'CmdOrCtrl+K',
+          },
+        ],
+      });
+    }
+
     this.menu = Menu.buildFromTemplate(template);
 
     Menu.setApplicationMenu(this.menu);
-  }
+  };
 
   openGettingStartedDocs = () => {
     shell.openExternal(`${baseRepoUrl}/docs/getting-started.md`);
```

Comment:
This isn't your change but this URL is not working. It's supposed to be `https://github.com/joshwcomeau/guppy/blob/master/docs/getting-started.md`

Reply:
Actually, I'll send a PR since it's unrelated to your stuff.